src/org/traccar/model/Position.java

```diff
@@ -88,6 +90,10 @@
 
     public static final String KEY_DRIVER_UNIQUE_ID = "driverUniqueId";
 
+    public static final String KEY_CELL_NET_CODE = "cellNetCode";
```

Comment:
Please explain new attributes.

Reply:
Inside the Laipac AVRMC message, there are two part included called Cell ID 1 and Cell ID 2. These to part contain info about the simcard and network inside the device. Cell ID 1 is split up in two parts, the first 4 bytes telling about the Cell Network Code (this is a code which is dependent on the cell network you are connected to). And the last 4 bytes which are the Cell ID Code (which are the codes connected to the simcard itself)

Reply:
Cell information has to be decoded into network object.